src/substrate-service.ts

```diff
@@ -66,6 +66,37 @@ export class SubstrateService {
     });
   }
 
+  public async issueAssetToUserBatch(destinationAccountPublicKey: string, amount: string, key_2: string, key_3: string): Promise<IssueAssetResponse> {
```

Comment:
@nikohosn 
Please, take a look to Cere's code guidelines. There you can find code conventions for variables, classes and etc.

Reply:
Understood, resolved, and removed the ambiguous namings